src/moves/MultiParticle.h

```diff
@@ -423,16 +431,11 @@ inline void MultiParticle::Accept(const uint rejectState, const ulong step)
   // Here we compare the values of reference and trial and decide whether to
   // accept or reject the move
   double MPCoeff = GetCoeff();
-  double delta_energy = sysPotNew.boxEnergy[bPick].real - sysPotRef.boxEnergy[bPick].real;
-  delta_energy += sysPotNew.boxEnergy[bPick].inter - sysPotRef.boxEnergy[bPick].inter;
-  delta_energy += sysPotNew.boxEnergy[bPick].recip - sysPotRef.boxEnergy[bPick].recip;
-  double uBoltz = exp(-BETA * delta_energy);
-
+  double uBoltz = exp(-BETA * (sysPotNew.Total() - sysPotRef.Total()));
   double accept = MPCoeff * uBoltz;
-  double pr = prng();
+  double pr = r123wrapper.GetRandomNumber(newCOMs.Count());
```

Comment:
Is there any reason we dont want to use Mersenne Twister (`prng()`) here? Since `newCOMs.Count() ` returns the number of molecule (not atoms), we are basically reusing same random number twice.

Reply:
The molecule indices are numbered 0 - newCOMs.Count() - 1, so this is really a different one from what was used before. Unless I missed something. I was testing some things when I made this change. I can change it back, but I figured in the long run we want o switch to the same RNG for everything, so...

Reply:
Looks good. I thought we draw random number for every atom and not molecule. I was wrong though. Up to you if you want to use `prng()` or random123